src/components/encryption.ts

```diff
@@ -107,7 +112,7 @@ export class EncryptedEventBroker {
     // We should probably make these LRUs eventually
     private eventsPendingAS: WeakEvent[] = [];
 
-    private syncingClients = new Set<any>();
+    private syncingClients = new Map<string, any>();
```

Comment:
We now store the userId => client. The client from matrix-js-sdk doesn't have a type :(